modules/bullet/btRayShape.cpp

```diff
@@ -39,7 +39,6 @@
 */
 
 btRayShape::btRayShape(btScalar length) :
-		btConvexInternalShape(),
```

Comment:
Why do you want to remove parent constructor call?

I saw that in the Joints too, can you please explain that?

Reply:
It is unnecessary, because child constructor automatically call public parent constructor without any parameters
```
#include<iostream>

using namespace std;

class basic {
  public: basic() {
    cout << "Basic\n";
  }
};
class child1: public basic {
  public: child1() {
    cout << "Child1\n";
  }
};
class child2: public basic {
  public: child2(): basic() {
    cout << "Child2\n";
  }
};

int main() {
  child1 c1;
  child2 c2;
}
```
This code prints same output to each objects
```
Basic
Child1
Basic
Child2
```

Reply:
Yes, sure. But unless it causes a specific problem I would like to call parent constructors explicitly for better readability.

Reply:
> Yes, sure. But unless it causes a specific problem I would like to call parent default constructors explicitly for better readability.